src/messaging/ExchangeContext.h

```diff
@@ -125,7 +125,8 @@ class DLL_EXPORT ExchangeContext : public ReferenceCounted<ExchangeContext, Exch
      *  @retval  #CHIP_NO_ERROR                             if the CHIP layer successfully delivered the message up to the
      *                                                       protocol layer.
      */
-    CHIP_ERROR HandleMessage(const PacketHeader & packetHeader, const PayloadHeader & payloadHeader, System::PacketBuffer * msgBuf);
+    CHIP_ERROR HandleMessage(const PacketHeader & packetHeader, const PayloadHeader & payloadHeader,
```

Comment:
The comment for this function _should_ change, though, since the function signature changed

Reply:
Ah, I misunderstood you as suggesting that none should change because `PacketBufferHandle` counts as a pointer.